pcdsdevices/spectrometer.py

```diff
@@ -218,6 +221,35 @@ def __init__(self, prefix, *, name, prefix_focus, prefix_energy, **kwargs):
                          prefix_energy=prefix_energy, **kwargs)
 
 
+class VonHamosCrystal_2(BaseInterface, GroupDevice):
+    """
+    Translation, rotation and tilt motors for control of a single crystal of
+    the MFX 6-crystals spectrometer.
+    """
+    tab_component_names = True
+
+    x = Cpt(BeckhoffAxis, ':X', kind='normal')
+    rot = Cpt(BeckhoffAxis, ':ROT', kind='normal')
+    tilt = Cpt(BeckhoffAxis, ':TILT', kind='normal')
+
+
+class VonHamos6Crystal(BaseInterface, GroupDevice):
+    """ MFX 6-crystal VonHamos spectrometer """
+    tab_component_names = True
+
+    c1 = Cpt(VonHamosCrystal_2, ':C1', kind='normal')
+    c2 = Cpt(VonHamosCrystal_2, ':C2', kind='normal')
+    c3 = Cpt(VonHamosCrystal_2, ':C3', kind='normal')
+    c4 = Cpt(VonHamosCrystal_2, ':C4', kind='normal')
+    c5 = Cpt(VonHamosCrystal_2, ':C5', kind='normal')
+    c6 = Cpt(VonHamosCrystal_2, ':C6', kind='normal')
+
+    rot = Cpt(BeckhoffAxis, ':ROT', kind='normal')
+    t1 = Cpt(BeckhoffAxis, ':T1', kind='normal')
```

Comment:
I feel like "t#" is pretty ambiguous here, since we've seen both "translation" and "tilt" in this context.  If scientists want this to be short, could we add something in the `doc` for these components letting us know what these are meant to control?  Since there's nothing in the docstring here either.  

Also it's entirely unclear where each of these axes lie. (Is one of these parallel to beam? Is it at some arbitrary angle?) Maybe that's not possible to know apriori, but if we can I think it warrants a bit more verbosity.

Reply:
That's a good point. I have changed the name to match the names the scientists gave in the screen.